schema/google/showcase/v1beta1/messaging.proto

```diff
@@ -289,7 +289,10 @@ message Blurb {
   option (google.api.resource) = {
     type: "showcase.googleapis.com/Blurb"
     pattern: "rooms/{room_id}/blurbs/{blurb_id}"
+    pattern: "rooms/{room_id}/blurbs/{blurb_id}.{legacy_room_id}"
     pattern: "user/{user_id}/profile/blurbs/{blurb_id}"
+    pattern: "user/{user_id}/profile/blurbs/{blurb_id}~{legacy_user_id}"
```

Comment:
A section on Pattern Uniqueness is being added to the AIP and what we have proposed here violates the uniqueness principle: https://github.com/googleapis/aip/pull/498/files#diff-5dab22c31f3732f8150a18b7eeba7515

We should change the new, compound patterns to something else so that they are valid. @chrisdunelm any ideas?

Reply:
Yes, this needs changing so every pattern is unique according to the uniqueness definition soon to be in AIP-123, as linked by @noahdietz.

I don't know the data model used by showcase, so not sure I can make any sensible suggestion about what to use.
Perhaps `rooms/{room_id}/blurbs/{blurb_id}/legacy/{legacy_room}.{legacy_blurb}` - although this may not make any sense!

Also, not to change now, but all these `_id` suffixes shouldn't really be here, as specified in AIP-123!

Reply:
Thanks Chris, that suggestion helps. Maybe we should move the generated {blurb_id} out to the compound section, making the "legacy" a sub-collection of the parent's "blurbs" collection, like so:

```suggestion
    pattern: "rooms/{room_id}/blurbs/legacy/{legacy_room_id}.{blurb_id}"
    pattern: "user/{user_id}/profile/blurbs/{blurb_id}"
    pattern: "user/{user_id}/profile/blurbs/legacy/{legacy_user_id}~{blurb_id}"
```

> all these _id suffixes shouldn't really be here, as specified in AIP-123!

Great point. I made #376 to track this!

Reply:
Thanks for the advice, making the changes now.